### PR TITLE
Created contact form page during pair programming

### PR DESCRIPTION
**Pull request by mnaibei**
Created contact form
Got styling advice from partners
Implemented changes
Partners committed some changes
Created PR

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -496,3 +501,104 @@ h2 {
...
 }
+
+/* contact form  */
+.contact {
+  display: flex;
+  flex-direction: column;
+  background-image: url("images/contact\ form\ shapes-mobile@2x.png"), url("images/Contact-me.png");
+  background-repeat: no-repeat;
+  background-position: right, center;
+  margin-top: 100px;
```
Comment (tugsem) [inline styles.css:513]: - [ ] Please decrease the margin as it is shorter in the figma design:
image
figma: 
image

File: styles.css [hunk 2]
```diff
@@ -496,3 +501,104 @@ h2 {
...
+
+.submit {
+  margin-top: 20px;
+  color: #396df2;
+  height: 60px;
+  width: 180px;
+  border-radius: 8px;
+  background-color: white;
+  border: 2px solid #6070ff;
+}
```
Comment (tugsem) [inline styles.css]: - [ ] Please remove the white space under the submit button. Also, decrease the padding of the button to match the figma design.
image
figma:
image
- [ ] One more thing, kindly add these button interactions in the figma design to make it more enhancing.
image

File: styles.css [hunk 3]
```diff
@@ -496,3 +501,104 @@ h2 {
   align-items: center;
   justify-content: space-between;
 }
+
```
Comment (tugsem) [inline styles.css:505]: - [ ] Please remove horizontal scrolling from your body to make it more professional and match the figma design.
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hi @mnaibei, @francis450, @vikas924, 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️Linters
✔️GitHub flow
✔️Professional documentation
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @mnaibei , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**